app/controllers/questions_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @question.increment_views
```

Comment:
Interesting, I didn't know ActiveRecord has such method. Cool. I will change that